Sources/HaCWebsiteLib/ViewModels/NotFound.swift

```diff
@@ -0,0 +1,27 @@
+import HaCTML
+
+struct NotFound {
+  
+  var node: Node {
+    return UI.Pages.base(
+      title: "Page Not Found",
+      content: Fragment(
+        El.Div[Attr.className => "NotFound_container"].containing(
+          El.Img[
+            Attr.className => "LandingIntroduction__titleImage",
+            Attr.src => "/static/images/hac-logo-dark.svg",
+            Attr.alt => "Hackers at Cambridge"
+          ],
+          El.Div[Attr.className => "LandingIntroduction__tagLine"].containing("Cambridge's student tech society"),
+          El.H1[Attr.className => "NotFound_title"].containing("404"),
+          El.P.containing("We can't find the page you're looking for..."),
+          El.A[Attr.href => "/"].containing(
+            El.Div[Attr.className => "LandingIntroduction__callToActionButton"].containing(
```

Comment:
Similar to the comments above, would it make sense to break this out into, for example, a `BigButton` block?